src/client/components/header.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react'
+
+class Header extends Component {
```

Comment:
We need to make a stylistic decision.

I have been using pure components, but you have been using the class syntax.

We need to choose one or the other